gramps/plugins/importer/importcsv.py

```diff
@@ -373,6 +377,24 @@ def parse(self, filehandle):
             LOG.debug(msg)
             LOG.debug("New Families: %d" % self.fam_count)
             LOG.debug("New Individuals: %d" % self.indi_count)
```

Comment:
Can you also add a new debug line for Places?

`LOG.debug("New Places: %d" % self.place_count)`

Reply:
Done

Reply:
@prculley 🌟🌟🌟🌟 Thank you.